report_thesis/src/references.bib

```diff
@@ -595,3 +595,10 @@ @misc{duan_ngboost_2020
 	keywords = {Computer Science - Machine Learning, Statistics - Machine Learning},
 	annote = {Comment: Accepted for ICML 2020},
 }
+
+@article{zemel_gbr,
+	title = {A {Gradient}-{Based} {Boosting} {Algorithm} for {Regression} {Problems}},
+	abstract = {In adaptive boosting, several weak learners trained sequentially are combined to boost the overall algorithm performance. Recently adaptive boosting methods for classification problems have been derived as gradient descent algorithms. This formulation justifies key elements and parameters in the methods, all chosen to optimize a single common objective function. We propose an analogous formulation for adaptive boosting of regression problems, utilizing a novel objective function that leads to a simple boosting algorithm. We prove that this method reduces training error, and compare its performance to other regression methods.},
+	language = {en},
+	author = {Zemel, Richard S and Pitassi, Toniann},
+}
```

Comment:
Wondering if it's better to use the original paper? [Greedy Function Approximation: A Gradient Boosting Machine](https://jerryfriedman.su.domains/ftp/trebst.pdf)

It's also the reference used for the sklearn implementation, which we use.

Reply:
And if you really don't want to read the original work, use, e.g., section 10.10.2 in the great Hastie's [Elements of Statistical Learning](https://hastie.su.domains/Papers/ESLII.pdf). 

IIRC, GBRT minimizes squared error via $\frac{1}{2}\[y_i-f(x_i)\]^2$, which doesn't necessitate calculating gradients, just the residuals $y_i-f(x_i)$.